content/references/protocol-reference/data-types/nftoken.md

```diff
@@ -112,36 +112,36 @@ The example sets three flags: lsfBurnable (0x0001), lsfOnlyXRP (0x0002), lsfTran
 
 ### TransferFee
 
-The TransferFee value specifies the percentage fee, in units of 1/10,000, charged by the issuer for secondary sales of the token. Valid values for this field are between 0 and 50,000, inclusive. A value of 1 is equivalent to 1bps or 0.01%, allowing transfer rates between 0% and 50%.
+The `TransferFee` value specifies the percentage fee, in units of 1/10,000, charged by the issuer for secondary sales of the token. Valid values for this field are between 0 and 50,000, inclusive. A value of 1 is equivalent to 1bps or 0.00001%, allowing transfer rates between 0% and 50%.
 
 
 ### Example
 
-This value sets the transfer fee to 314 bps, or 3.14%.
+This value sets the transfer fee to 314, or .00314%.
 
 ![Txr Fee](img/nftokenb.png "Txr Fee")
```

Comment:
Just noticed, this might be the only place the docs use "txr" to abbreviate "transfer".

```suggestion
![Transfer Fee](img/nftokenb.png "Transfer Fee")
```